src/app/components/navigation/navigation.component.ts

```diff
@@ -45,4 +50,9 @@ export class NavigationComponent implements OnInit {
   public setLanguage(lang: string) {
     this.translateService.use(lang);
   }
+
+  private isValidLanguage(lang: string) {
+    const validLanguages = [ 'en', 'fr', 'sv', 'da' ];
```

Comment:
In PR #165 (not yet merged) I update all places where the language tags are mentioned to be in alphabetical order to have it consistent. I suggest that they are listed as
```
    const validLanguages = [ 'da', 'en', 'fr', 'sv' ];
```

Reply:
Fixed